platform/android/buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
+    gradlePluginPortal()
+    google()
+    mavenCentral()
+    maven("https://plugins.gradle.org/m2/")
+}
+
+dependencies {
+    implementation("com.github.spotbugs.snom:spotbugs-gradle-plugin:5.2.1")
```

Comment:
What is this?

Reply:
this is from the sample I started working with, will remove it.

Reply:
Thank you!

Reply:
@louwers I have pushed new changes, plz run the tests